xmlschema/__init__.py

```diff
@@ -54,3 +55,12 @@
     'XsdGlobals', 'XMLSchemaBase', 'XMLSchema', 'XMLSchema10', 'XMLSchema11',
     'XsdComponent', 'XsdType', 'XsdElement', 'XsdAttribute',
 ]
+
```

Comment:
The translation install should be activable by option or by an utility function after xmlschema import.
This because someone might not want to turn on translation or might want to use another custom translation.

Reply:
Yes, I think about some settings option or environment variable. But all strings in code must be wrapped in `_()` function, which initializing in `translation.install()`. And I don't know how make translation optionable. Maybe language code should be in settings.

Reply:
This is a library and currently it doesn't have settings, and I would prefer to avoid this in order to keep it simple.

As i saw installed translations are stacked into gettext module internals, so i think that a reasonable manner to handle this is to provide an utility to add package translations, also parametrized with language codes. I don't know how currently, this could be decided after looking at solutions used in other packages.

Reply:
Add initialization via `schema = xmlschema.XMLSchema11(xsd, use_translation=True)`